config/jobs/kubernetes/sig-cloud-provider/gcp/upgrade-gce.yaml

```diff
@@ -1,7 +1,7 @@
 periodics:
 # GCE upgrades
 - interval: 2h
-  name: ci-kubernetes-e2e-gce-stable1-beta-upgrade-cluster
+  name: ci-kubernetes-e2e-gce-stable1-master-upgrade-cluster
```

Comment:
you use latest everywhere else instead of master

Reply:
Same in couple other places

Reply:
fixed